python/python_fastapi_server/routers/edr.py

```diff
@@ -391,7 +402,9 @@ def get_collectioninfo_for_id(
             crs=crs,
             output_formats=output_formats,
         )
-    return collection
+
+    get_cap = get_capabilities(edr_collection)
+    return collection, get_cap["expires"]
```

Comment:
It's bit confusing, one function return the `expires` (why not `ttl`) as part of a dict, others use a tuple.

Let try to avoid mixing content and TTL info, so always use tuples for TTL?

Reply:
get_capabilties now returns a tuple (with expires)